pkg/storage/toml.go

```diff
@@ -16,35 +16,16 @@ type TOMLStore struct {
 	projects map[string]config.Project
 }
 
-// New creates a new instance to red the configuration from a toml file.
```

Comment:
As we are not using Toml to store configurations, could be remove this type completely?

Reply:
Fixed in commit 724a421d492646da164328fc0b05d42eeaf5893c